### PR TITLE
Moving personal details to top

### PR DESCRIPTION
**Pull request by B-Bonner**
Preview link: https://deploy-preview-431--bat-design-history.netlify.app/apply-for-teacher-training/moving-personal-details-to-top/

### CODE REVIEW

**Review by frankieroberto — COMMENTED** [review]
Looks good! Made some suggestions.

Should we also acknowledge somewhere that many candidates may _start_ from Find, rather than Apply, and so picking their first course at least will still happen before entering the personal details?

**Review by EmmaFrith — APPROVED** [review]
looks good @B-Bonner!

File: app/posts/apply-for-teacher-training/2021-08-17-moving-personal-details-to-top.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+
+Previously, the Courses section was at the top of the application. This was to help direct candidates to the right site to complete their application. Some courses were on UCAS and others were on Apply for teacher training.
+
+But when UCAS is switched off in September, candidates will be able to apply for all courses using Apply.
+
+### Getting candidates to add their personal details before choosing a course
```
[the remark this answers is not in the document]
Comment (B-Bonner) [reply app/posts/apply-for-teacher-training/2021-08-17-moving-personal-details-to-top.md]: Yeah, I thought that too.

File: app/posts/apply-for-teacher-training/2021-08-17-moving-personal-details-to-top.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+When we worked on the visa sponsorship content on Apply, we thought it would be helpful to warn candidates who need visas if they chose a course that does not sponsor visas. That way they could change their course choice before submitting their application.
```
[the remark this answers is not in the document]
Comment (B-Bonner) [reply app/posts/apply-for-teacher-training/2021-08-17-moving-personal-details-to-top.md]: Both good points, thanks!

File: app/posts/apply-for-teacher-training/2021-08-17-moving-personal-details-to-top.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  hideContents: true,
+  items: [{
+    text: "Visa warning in the Course choices summary",
+    img: {
+      src: "visa-warning-message.png"
+    }
+  }]
+}) }}
+
+To do that, we needed them to enter their nationality and immigration status before they chose their courses. So we decided to move the Personal details section to the top of the application.
```
[the remark this answers is not in the document]
Comment (B-Bonner) [reply app/posts/apply-for-teacher-training/2021-08-17-moving-personal-details-to-top.md]: Yep, sounds good. Makes sense.

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**